src/CodeGen_Internal.h

```diff
@@ -67,13 +67,14 @@ Expr lower_euclidean_mod(Expr a, Expr b);
 Stmt unpredicate_loads_stores(Stmt s);
 
 /** Given an llvm::Module, set llvm:TargetOptions, cpu and attr information */
-void get_target_options(const llvm::Module &module, llvm::TargetOptions &options, std::string &mcpu, std::string &mattrs);
+void get_target_options(const llvm::Module &module, llvm::TargetOptions &options,
+                        std::string &mcpu, std::string &mattrs);
 
 /** Given two llvm::Modules, clone target options from one to the other */
 void clone_target_options(const llvm::Module &from, llvm::Module &to);
 
 /** Given an llvm::Module, get or create an llvm:TargetMachine */
-std::unique_ptr<llvm::TargetMachine> make_target_machine(const llvm::Module &module);
+ std::unique_ptr<llvm::TargetMachine> make_target_machine(const llvm::Module &module);
```

Comment:
Remove leading space. In general, there are a lot of changes in this PR that are just reformatting left over from the target plumbing that would be nice (but not necessary) to revert.